### PR TITLE
Added the activity suggestion app idea

### PR DESCRIPTION
**Pull request by magmukendi**
This is a new app idea I suggest. It would be great if you can add a hacktoberfest-accepted label to this.

Thanks for the opportunity.

### CODE REVIEW

**Comment by roopeshsn** [comment]
Are you expecting me to write the body of the post or what do you think about me maintaining this project @LeGrandMAG? Do you think that I have no job and sit in front of the computer to see this shit! Are you thinking it's a contribution?

File: db/all-ideas/app-ideas/randomactivity.mdx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+title: Create a Activity Suggestion App
+slug: randomactivity
+createdAt: Sun Oct 16 2022 23:41:57 GMT+0900 (Korean Standard Time)
+category: app-ideas
+summary: Activity suggestion app help bored users decide on what they want to randomly do on that day.
+tags: javascript, css
+author: LeGrandMAG
+---
+
+# Create a Activity Suggestion App
```
Comment (roopeshsn) [inline db/all-ideas/app-ideas/randomactivity.mdx:11]: 😞

**Review by roopeshsn — COMMENTED** [review]
Today I felt to leave the project unmaintained

File: db/all-ideas/app-ideas/randomactivity.mdx (hunk 1 above)
Comment (magmukendi) [reply db/all-ideas/app-ideas/randomactivity.mdx:11]: Your instructions don't mention anything about the body of the markdown file.

When I run the app locally, it prompts me about the title, the author, the smug, the tags, and the summary, but it does not say anything about the body. I think there should be some more instructions regarding completing the body of the markdown file, or when the user runs plop, let them know that they should complete the body (description) of their app manually. 

I created an issue about that here #20

**Comment by magmukendi** [comment]
@roopeshsn, let me know what is wrong so that I can fix it.

**Pull request closed without merging**